src/utils/SpokeUtils.ts

```diff
@@ -393,3 +395,7 @@ export async function findFillBlock(
 export function isZeroAddress(address: string): boolean {
   return address === ZERO_ADDRESS || address === ZERO_BYTES;
 }
+
+export function getMessageHash(message: string): string {
+  return isMessageEmpty(message) ? ZERO_BYTES : keccak256(message);
```

Comment:
Instead of ZERO_BYTES should this be ethers.constants.HashZero like I use here? https://github.com/across-protocol/sdk/blob/pre-fills/src/constants.ts#L60 

I think both are ok but maybe better to be consistent

Reply:
Ah - we're exporting the same constant multiple times. We're also exporting `HashZero` as `ZERO_BYTES`, and `ZERO_BYTES` is what's used in `isZeroAddress()` (see line 396 in the diff above).

https://github.com/across-protocol/sdk/blob/2349b73a03d9ba62a8c52328255da6bf7ab48f78/src/constants.ts#L13

`ZERO_BYTES` might be more "portable" because `bytes32(0)` is also used in the context of the zero address within `isZeroAddress()`. wdyt?

Reply:
ah ok i'll change my PR to use ZERO_BYTES then